Jenkinsfile

```diff
@@ -10,7 +10,8 @@ pipeline {
 
     options {
       timestamps ()
-	  lock(resource: 'sirius-desktop-tests')
+      lock(resource: 'sirius-desktop-tests')
+      timeout(time: 120, unit: 'MINUTES')
```

Comment:
I have explicitly removed this kind of timeout in commit [5bec3484](https://github.com/eclipse-sirius/sirius-desktop/commit/5bec34841cb72ed68a0e8244a19c25e85f5ad643). The reason is explained in the comment of this commit:
```
[releng] Remove global 1 hour timeout
When several PRs are pushed simultaneously, each PR can occupy part of the 4 free slots.
So parallel sub-stages can take several minutes to actually start (even if the timeout starts
at the start of the "parent" stage).
FYI, a timeout already exists individually on each sub-stage (currently set at 2 hours).
```
Do you have any comments/contraindications to what I indicated at the time?

Reply:
Thanks for the insight. I missed the timeouts on the stages level. 

When I killed https://ci.eclipse.org/sirius/job/sirius-pr-check/view/change-requests/job/PR-397/1/, the parent job was running for 17 hours, so the stage timeouts do not seem to cover all cases. I would recommend to still set a global timeout (it can be more than 120 minutes, if required), to avoid blocking build resources for many hours.

Reply:
OK. I propose to set the global timeout to 360. If after 6 hours the three sub part are not finished we can seriously consider that something went wrong.

Can you change the value in your PR?
Can you also:
- Add [releng] prefix to your commit message
- Add details in the commit message to keep a summary of our exchanges, something like: `The global timeout is set to 6 hours. In commit [5bec3484](https://github.com/eclipse-sirius/sirius-desktop/commit/5bec34841cb72ed68a0e8244a19c25e85f5ad643), the global timeout was removed because specific timeouts exist for substages. It was observed that in some circumstances, this is not enough. So a "big" global timeout is added to handle this case.`